packages/apple-pay-integration/e2e/ApplePaySessionPaymentStepMockObject.ts

```diff
@@ -0,0 +1,137 @@
+const addApplePaySessionToChrome = () => {
+    class ApplePaySessionPaymentStep implements ApplePaySession {
+        version: number;
+        paymentRequest: ApplePayJS.ApplePayPaymentRequest;
+
+        static STATUS_SUCCESS = 1;
+        static STATUS_FAILURE = 2;
+
+        static supportsVersion(_versionNumber) {
```

Comment:
Need type definition for all parameters in this file.

Reply:
Sure thing I can update that